src/Symfony/Component/VarDumper/Test/VarDumperTestTrait.php

```diff
@@ -19,6 +19,23 @@
  */
 trait VarDumperTestTrait
 {
+    private static $casters = [];
+    private static $varCloner;
+    private static $flags;
+
+    protected static function setUpVarDumper(array $casters, int $flags = null): void
```

Comment:
why did you choose to use the static API?

Reply:
So this can be used once in `setUpBeforeClass/tearDownAfterClass` if your whole test class uses a single set of specific casters.

Reply:
:+1: